BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -17,6 +17,7 @@ public struct ScrollView: Element {
     public var showsHorizontalScrollIndicator: Bool = true
     public var showsVerticalScrollIndicator: Bool = true
     public var pullToRefreshBehavior: PullToRefreshBehavior = .disabled
+    public var keyboardDismissMode: UIScrollView.KeyboardDismissMode = .none
```

Comment:
This matches the UIKit default, but it also is almost nearly wrong... Consider `.interactive`?

Reply:
Leaving as is.